### ISSUE TITLE
Lack of focus trap in hamburger menu after the last item in the first panel of the hamburger menu

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#35]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
[Aan de slag met API's](https://don.apps.digilab.network/kennisbank/apis/aan-de-slag/)

**Probleembeschrijving:**
Na het laatste zichtbare item in het openingspaneel van het Hamburger Menu (met de items die oorspronkelijk in de Aside stonden), verlaat de focus het hamburger menu en gaat door items op de achtergrond. De gebruiker moet óf achteruit tabben óf door elk afzonderlijk item op de webpagina tabben om weer terug te komen in het hamburger menu.

**Referentiecode:**
`<button aria-label="Categorie zijbalk uitklappen 'Tools'" aria-expanded="false" type="button" class="clean-btn menu__caret"></button>`

**Gerelateerde WCAG:**
WCAG 2.4.3 Focus order

**Referentieafbeelding:**
<img width="1182" alt="Image" src="https://github.com/user-attachments/assets/5e307e6a-370a-4fae-b858-7258477041c5" />

_**Referentienummer:** A046_
